Makefile

```diff
@@ -36,6 +36,7 @@ clean:
 
 lint: lint-js lint-python
 
+# TODO (joshuarli): move flake8 to pre-commit, keep backwards compat for lint-python
 lint-python:
```

Comment:
Ideally we should change all our lint-python stuff to run pre-commit.

Reply:
That's not semantically correct though. lint-python should be kept, but it should only run flake8 via pre-commit.

Reply:
actually. We need this plugin

https://pypi.org/project/flake8-black/

We do something similar for our eslint in javascript.

Reply:
see discussion here: https://github.com/getsentry/getsentry/pull/3087#discussion_r308489363